content/en/distributed-systems.md

```diff
@@ -2,11 +2,10 @@
 title: Distributed System
 status: Completed
 category: concept
-tags: ["architecture", "", ""]
+tags: ["architecture", "infrastructure", "fundamental"]
 ---
 
-A distributed system is a collection of autonomous computing elements 
-connected over a network that appears to users as a single coherent system. 
+A distributed system is a collection of autonomous computing elements connected over a network that appears to users as a single coherent system. 
```

Comment:
For reviewers: no content update made here, just a semantic line break adjustment.